manifests/00-namespace.yaml

```diff
@@ -1,4 +1,6 @@
 apiVersion: v1
 kind: Namespace
 metadata:
+  annotations:
+    openshift.io/node-selector: ""
```

Comment:
I do not understand, why this should be empty string. As I understand the api-server-operator PR you linked will ensure that any default node selector specified for cluster is copied to project. But I do not understand how setting this to empty string helps. @ravisantoshgudimetla can you clarify?